battleground-state-changes.html.tmpl

```diff
@@ -320,7 +320,8 @@
                     if (latestContent.updates.states_updated.length) {
                         Array.from(document.getElementsByTagName("table")).forEach(table => {
                             let newTable = latestContent.dom.getElementById(table.id);
-                            table.parentNode.replaceChild(newTable, table);
+                            table.replaceChild(newTable, table);
```

Comment:
Looks like this change needs to be reverted.

Reply:
Sure does.